### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by mirjoy**
Added new "favorites" join table - tests commented out for now so we run the migration before adding the  giant seed data file

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
   post '/login',    to: 'sessions#create'
   get '/logout', to: 'sessions#destroy'
 
+  resources :favorites
```
Comment (treyx) [inline config/routes.rb]: Do we need all these routes right now? Maybe we should limit this resource.

Comment (mirjoy) [inline config/routes.rb]: We're going to need most of them - not update or edit, so I could remove those.

**Comment by worace** [comment]
awesome

File: spec/feature/authenticated_traveler_spec.rb [hunk 2]
```diff
@@ -84,4 +83,16 @@
     expect(page).to have_content("You must be a host to see your guests")
     expect(current_path).to eq(root_path)
   end
+
+  xit "can add and remove properties from a wish list" do
```
Comment (worace) [inline spec/feature/authenticated_traveler_spec.rb]: any other tests?

**Pull request merged**